docs/DeveloperGuide.md

```diff
@@ -36,3 +33,115 @@
 ## Instructions for manual testing
 
 {Give instructions on how to do a manual product testing e.g., how to load sample data to be used for testing}
+
+
+<!-- Zhan hong working on this section-->
+# Review Feature
```

Comment:
Good detailed explanation showing how the features works, with simple and easily understand explanation and good use of the sequence diagrams to make it clearer, could included the termination to make it clearer